tests/test_config.py

```diff
@@ -0,0 +1,111 @@
+from conftest import upstream_webdav_server, tmp_certificate
+from flask import url_for
+import pytest
+from typing import Any
+
+
+@pytest.mark.timeout(30)
+def test_valid_owncert_config(app: Any, client: Any):
+    with upstream_webdav_server():
+        with app.app_context():
+            ca_bundle, certificate = tmp_certificate(1)
+            open(app.config['CTADS_CABUNDLE'], 'w').write(ca_bundle)
```

Comment:
it's just a bit weird that client code which this test emulates is writing server file. It's not a bit issue but a bit confusing.
In principle server should almost always have right CA and only in some test we check what happens if we use wrong CA to sign our key.
Just wondering if this can be made more clear with fixtures as I wrote in conftest